package.json

```diff
@@ -36,7 +36,7 @@
     "keycloak-angular": "^4.0.2",
     "keycloak-js": "^4.6.0",
     "ngx-mat-select-search": "^1.8.0",
-    "ngx-spinner": "^6.1.2",
```

Comment:
Why are we using an external plugin for spinner

Reply:
It is used in sl-portal-library